src/NServiceBus.Core.Tests/Sagas/ComponentTests/When_completing_a_saga_with_no_defined_correlation_property.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_correlation_property : SagaPersisterTests<SagaWithoutCorrelationProperty, SagaWithoutCorrelationPropertyData>
```

Comment:
We are using the no_correlation property vs correlation property lingo throughout all the tests. This for me is similar to https://github.com/Particular/NServiceBus/pull/4496#discussion_r113184708